modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -537,6 +537,63 @@ void ONNXImporter::populateNet(Net dstNet)
         {
             replaceLayerParam(layerParams, "size", "local_size");
         }
+        else if (layer_type == "InstanceNormalization")
+        {
+            if (node_proto.input_size() != 3)
+                CV_Error(Error::StsNotImplemented,
+                         "Expected input, scale, bias");
+
+            layerParams.blobs.resize(2);
+
+            if (!node_proto.input(1).empty()) {
```

Comment:
If `node_proto.input_size() == 3` can we have an empty input name?

Reply:
In my experience no. However the same thing happens for the BatchNorm layer, and I was just trying to follow the same convention, also felt like an extra sanity check wouldnt hurt. Let me know if I should remove it

Reply:
Let's assume that they aren't empty.

Reply:
@dkurt, should be fixed now